sdk-setup/src/mer-sdk-chroot

```diff
@@ -396,6 +403,9 @@ setarch i386 chroot ${sdkroot} /bin/su -s /bin/bash -l $user -- -c "$(cat <<END
 export SSH_AUTH_SOCK='${MY_SSH_AUTH_SOCK}'
 export SSH_AGENT_PID=${SSH_AGENT_PID:-}
 export MERSDK='$sdkroot'
+export GNUPGHOME='${HOMEDIR}/${SFDK_GNUPGHOME_DIR}'
+
+[[ ! -d "${HOMEDIR}/${SFDK_GNUPGHOME_DIR}" ]] && mkdir -p "${HOMEDIR}/${SFDK_GNUPGHOME_DIR}" && sudo chmod 700 ${HOMEDIR}/${SFDK_GNUPGHOME_DIR}
```

Comment:
I would create it only when we need it (from mb2), so that we do not pollute user's home unnecessarily.

This answers your question "Should we take care of the existence of this folder or leave it to the user's discretion as well?" I believe.

Reply:
I agree, but if you just run `gpg2 --import` (any gpg2 command), you will get an error saying that there is no `GNUPGHOME`. May be create `GNUPGHOME` under `/srv/mer/sdks/sfossdk/`? Then it will be deleted every time reinstall it

Reply:
yeah, maybe better use `$HOME/.config/SailfishSDK/gnupg`